CMakeLists.txt

```diff
@@ -5,22 +5,7 @@ if (CMAKE_VERSION VERSION_GREATER_EQUAL "3.24.0")
     cmake_policy(SET CMP0135 NEW)
 endif()
 
-include(FetchContent)
-
-if(IOS)
-    FetchContent_Declare(ios-cmake
-        GIT_REPOSITORY https://github.com/leetal/ios-cmake.git
-        GIT_TAG 04d91f6675dabb3c97df346a32f6184b0a7ef845)
-
-    message(STATUS "Fetching ios-cmake")
-    FetchContent_MakeAvailable(ios-cmake)
-    message(STATUS "Fetching ios-cmake - done")
-
-    set(CMAKE_TOOLCHAIN_FILE "${ios-cmake_SOURCE_DIR}/ios.toolchain.cmake" CACHE PATH "")
-    set(PLATFORM "OS64COMBINED" CACHE STRING "")
-    set(DEPLOYMENT_TARGET "12" CACHE STRING "")
-endif()
-
+include(content.cmake)
```

Comment:
See other comment.

Reply:
Let's talk offline

Reply:
include moved.

Reply:
Fixed by moving everyfetch content in root cmake.